### PR TITLE
#159965476 Receive descriptive signup/Registration validation errors.

### PR DESCRIPTION
**Pull request by ghost**
## What does this PR do?
```
Add descriptive error message when for invalid user parameters in an attempt to signup.

Invalid user parameters include:
- existing user email
- invalid email
- password of length less than 8
- password not being alphanumeric

```

## Description of Task to be completed?
```

1. Customize error messages in the Registerserializer
2. Add tests 

```

## How should this be manually tested?
```

1. Clone the repository
2. Check out the branch ft-register-validation-159965476
3. Run the command ` coverage run --source=authors/apps manage.py test && coverage report -m`

```

## What are the relevant pivotal tracker stories?

[#159965476](https://www.pivotaltracker.com/story/show/159965476)

## Screenshots if available to support PR.
Invalid user parameters
not available
invalid email
password of length less than 8
alphanumeric

Testing 
coverage report

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    */authors/apps/authentication/migrations/0001_initial.py
+    */authors/apps/authentication/migrations/__init__.py
+    */authors/apps/profiles/__init__.py
+    */authors/apps/profiles/tests/__init__.py
+    */authors/apps/authentication/test/__init__.py
+    */authors/apps/authentication/test/base.py
+    */authors/apps/authentication/test/test_auth.py
+    */authors/apps/authentication/test/test_models.py
+    */authors/apps/authentication/test/test_register_validation.py
+    */authors/apps/base_test.py
```
Comment (Quantum-35) [inline .coveragerc:16]: Good implementation omitting the __init__.py

**Review by Quantum-35 — APPROVED** [review]
Looks good to me

**Review by Quantum-35 — APPROVED** [review]

**Review by Derrickkip — APPROVED** [review]
LGTM

**Review by WNjihia — APPROVED** [review]
Good job on this 👍

**Pull request merged**